SimPEG/Regularization.py

```diff
@@ -394,13 +426,6 @@ def __init__(self, mesh=None, **kwargs):
     cell_weights = properties.Array(
         "regularization weights applied at cell centers", dtype=float
     )
-    regmesh = properties.Instance(
-        "regularization mesh", RegularizationMesh, required=True
-    )
-    mapping = properties.Instance(
-        "mapping which is applied to model in the regularization",
-        Maps.IdentityMap, default=Maps.IdentityMap()
```

Comment:
The default should not be instantiated here in future. That means the default is shared between *all* instances of that class.

Instead we should use: `default=Maps.IdentityMap` not `default=Maps.IdentityMap()` if the default is a callable it is instantiated, for more complicated things, we can use a `lambda` function.